tests/e2e/specs/header.cy.js

```diff
@@ -15,19 +15,63 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 describe('Header Component', () => {
-  it('test username@host', () => {
-    cy
-      .visit('/#/')
-      .get('.c-environment-info')
-      .should('exist')
-      .should('be.visible')
-      .get('#username.v-chip')
-      .should('exist')
-      .should('be.visible')
-      .contains('user')
-      .get('#host.v-chip')
-      .should('exist')
-      .should('be.visible')
-      .contains(/localhost:\d{4,5}/)
+  it('Displays read-only server info in single-user mode', () => {
+    cy.visit('/#/')
+      .get('#cylc-owner-combobox')
+      .should('be.disabled')
+      .get('#cylc-deployment-combobox')
+      .should('be.disabled')
+      .get('[data-cy=multiuser-go-btn]')
+      .should('not.exist')
+  })
+})
+
+describe('Header Component multiuser', () => {
+  beforeEach(() => {
+    cy.intercept('/userprofile', {
+      body: {
+        name: 'userTest',
+        groups: [
+          'cylc',
+          'developer'
+        ],
+        created: '2021-03-23T23:26:23.606Z',
+        admin: true,
+        server: '/user/cylc/',
+        permissions: [
+        ],
```

Comment:
Tests are failing because of the no-auth landing page you added
```suggestion
        permissions: [
          'read',
        ],
```